libs/ui/layout/src/lib/block-list/block-list.component.spec.ts

```diff
@@ -147,4 +147,27 @@ describe('BlockListComponent', () => {
       expect(component.isLastPage).toBe(false)
     })
   })
+
+  describe('set initial height as min height, keeps value when height changes', () => {
+    beforeEach(() => {
+      Object.defineProperties(component.blockContainer.nativeElement, {
+        clientHeight: {
```

Comment:
we need to do this because jsdom doesn't actually compute the width and height of things; everything is always 0 in tests